### PR TITLE
Updated properties, removed reference to efiling

### PR DESCRIPTION
**Pull request by KDNeufeld**
# Description

This PR includes the following proposed change(s):

Replaced "efiling" content in properties with "ai".

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/ai-reviewer-api/jag-reviewer-api.yaml [hunk 1]
```diff
@@ -2,14 +2,14 @@ openapi: 3.0.0
 info:
   version: 0.7.0
   title: jag-reviewer-api
-  description: Efiling AI Reviewer api
+  description: AI Reviewer api
```
Comment (karthikmsqa) [inline src/backend/ai-reviewer-api/jag-reviewer-api.yaml:5]: Should be 'Ai Reviewer api' for consistency?

File: src/backend/ai-reviewer-api/src/test/jag-reviewer-api.postman_collection.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
 	"info": {
 		"_postman_id": "45a8c23a-4549-4d89-9119-996ec7331777",
 		"name": "jag-reviewer-api Copy",
-		"description": "Efiling AI Reviewer api",
+		"description": "AI Reviewer api",
```
Comment (karthikmsqa) [inline src/backend/ai-reviewer-api/src/test/jag-reviewer-api.postman_collection.json:5]: same as above.

File: src/backend/ai-reviewer-api/jag-reviewer-api.yaml (hunk 1 above)
Comment (KDNeufeld) [reply src/backend/ai-reviewer-api/jag-reviewer-api.yaml]: Yeah, good.  Done.

File: src/backend/ai-reviewer-api/src/test/jag-reviewer-api.postman_collection.json (hunk 2 above)
Comment (KDNeufeld) [reply src/backend/ai-reviewer-api/src/test/jag-reviewer-api.postman_collection.json]: Yeah, good. Done.

**Review by karthikmsqa — APPROVED** [review]
Nice!

**Pull request merged**